README.md

```diff
@@ -770,6 +770,12 @@ Reactive twin of the `security/keycloak-oidc-client-extended`, extends `security
 
 - Verifies Proof Of Key for Code Exchange support for a Keycloak and Red Hat Single Sign-On together with OIDC Single Page Application logout flow
 
+### `security/keycloak-oidc-client-standalone`
+
+Verifying Dev Mode experience using `quarkus-oidc-client` extension without `quarkus-oidc` extension.
```

Comment:
I am not an English major, but I am not sure if present continuous should be used here. I'll leave it to you to decide.

Reply:
I would use `Verifies`, that's used in other sections